docs/index.md

```diff
@@ -45,72 +43,122 @@ Download the latest version of the CLI for your operating system and make sure t
     event-cli.exe help
     ```
 
-### 2. Pushing data to Pulse
+## 2. Pushing data to Pulse
 
-You must use the CLI or the webhook to send information to Pulse about the following key events whenever they happen in your software delivery workflow:
+To measure the performance of your team, you must send information to Pulse about the following key events whenever they happen in the software delivery workflow of your primary application or service:
```

Comment:
We now mention the "primary application or service". However, we'll probably be able to improve this once we understand a few real-life scenarios from our beta users.